src/control/lib/control/auto.go

```diff
@@ -516,8 +516,8 @@ func checkCPUs(log logging.Logger, numSSDs, coresPerNUMA int) (int, int, error)
 	return numTargets, calcHelpers(log, numTargets, coresPerNUMA), nil
 }
 
-func defaultIOSrvCfg(idx int) *ioserver.Config {
-	return ioserver.NewConfig().
+func defaultIOSrvCfg(idx int) *engine.Config {
```

Comment:
Doesn't need to be updated now, especially given that these are unexported, but these references to IOSrv should be updated.

Reply:
fixed in https://github.com/daos-stack/daos/pull/4611